include/sys/zfs_znode.h

```diff
@@ -275,7 +275,7 @@ extern void zfs_log_symlink(zilog_t *zilog, dmu_tx_t *tx, uint64_t txtype,
 extern void zfs_log_rename(zilog_t *zilog, dmu_tx_t *tx, uint64_t txtype,
     znode_t *sdzp, const char *sname, znode_t *tdzp, const char *dname,
     znode_t *szp);
-extern void zfs_log_write(zilog_t *zilog, dmu_tx_t *tx, int txtype,
+extern void zfs_log_write(zilog_t *zilog, dmu_tx_t *tx,
```

Comment:
While I agree that txtype argument is constant, you may see other functions around also have it.  I haven't looked whether other functions always have the same arguments or there are exceptions, but would be good to either change globally or not change only one function.